src/main/java/at/hannibal2/skyhanni/features/inventory/SuperCraftFeatures.kt

```diff
@@ -0,0 +1,26 @@
+package at.hannibal2.skyhanni.features.inventory
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.api.GetFromSackAPI
+import at.hannibal2.skyhanni.events.LorenzChatEvent
+import at.hannibal2.skyhanni.utils.DelayedRun
+import at.hannibal2.skyhanni.utils.NEUInternalName
+import at.hannibal2.skyhanni.utils.PrimitiveItemStack
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class SuperCraftFeatures {
+    private val regex = "§eYou Supercrafted §r§r§r§[^.*]([^x§]+?)(?:§r§8x(\\d+))?§r§e!".toRegex();
```

Comment:
you should use repopattern here so we can change it without a new skyhanni version if needed, check the [contributing docs](https://github.com/hannibal002/SkyHanni/blob/beta/CONTRIBUTING.md), could look something like 
```kotlin
private val craftedPattern by RepoPattern.pattern(
    "inventory.supercrafting.craft",
    "§eYou Supercrafted §r§r§r§[^.*]([^x§]+?)(?:§r§8x(\\d+))?§r§e!"
)
```

Reply:
done